DogsList/DogsList/Model/DogManager.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  DogManager.swift
+//  DogsList
+//
+//  Created by Nick Krasnitskiy on 06.07.2021.
+//
+
+import Foundation
+
+protocol DogManagerDelegate: class {
```

Comment:
Using 'class' keyword for protocol inheritance is deprecated; use 'AnyObject' instead

Reply:
Thanks. Ok.